TYPO3.Flow/Classes/TYPO3/Flow/Security/Account.php

```diff
@@ -347,6 +360,51 @@ public function setExpirationDate(\DateTime $expirationDate = null)
     }
 
     /**
+     * @return integer
+     * @api
+     */
+    public function getFailedAuthenticationCount()
+    {
+        return $this->failedAuthenticationCount;
+    }
+
+    /**
+     * @param integer $failedAuthenticationCount
+     * @return void
+     */
+    public function setFailedAuthenticationCount($failedAuthenticationCount)
+    {
+        $this->failedAuthenticationCount = $failedAuthenticationCount;
+    }
```

Comment:
instead of such a setter method, I'd rather like a "authenticationSuccessful()" method or s.th. like this, which sets the count to 0 and the lastSuccessfulAuthenticationDate to the current date.